cypress.config.ts

```diff
@@ -3,15 +3,13 @@ import { resetStubs } from './integration_tests/mockApis/wiremock'
 import stubs from './integration_tests/mockApis/stubs'
 
 export default defineConfig({
-  chromeWebSecurity: false,
```

Comment:
Removing because it doesn't seem this is needed? It only works in chrome and tests pass without it.